tests/tt_metal/tt_metal/perf_microbenchmark/1_compute_mm/test_compute_mm.cpp

```diff
@@ -177,10 +300,14 @@ int main(int argc, char** argv) {
             log_error(LogTest, "Command line arguments found exception", e.what());
         }
 
+        if (not single_core) {
+            TT_ASSERT(dtype == 0, "multi core test only supports bfp8_b");
+        }
+
         ////////////////////////////////////////////////////////////////////////////
         //                      Env and Device Setup
         ////////////////////////////////////////////////////////////////////////////
-        if (fast_dispatch_mode == false) {
+        if (not single_core and fast_dispatch_mode == false) {
```

Comment:
Should the single-core Matmul benchmark only run in the fast dispatch mode?
If so, I suggest this code.

```suggestion
        if (single_core) {
            TT_ASSERT(fast_dispatch_mode, "single core test only supports in fast dispatch mode");
        }
        
        if (fast_dispatch_mode == false) {
```